src/main/java/org/thief/repository/dao/Camera.java

```diff
@@ -0,0 +1,35 @@
+package org.thief.repository.dao;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+@Entity
+@Table(name = "cameras")
+@Getter
+@Setter
+public class Camera {
```

Comment:
Please consider to think what is your hashcode & equals contract for entity.

Reply:
Same as above